src/sensai/evaluation/eval_stats/eval_stats_base.py

```diff
@@ -21,13 +21,17 @@
 
 
 class EvalStats(Generic[TMetric]):
-    def __init__(self, metrics: List[TMetric]):
+    def __init__(self, metrics: List[TMetric], additionalMetrics: List[TMetric] = None):
         if len(metrics) == 0:
             raise ValueError("No metrics provided")
         self.metrics = metrics
+        # Implementations of EvalStats will typically provide default metrics, therefore we include
+        # the possibility for passing additional metrics here
+        if additionalMetrics is not None:
+            self.metrics.extend(additionalMetrics)
```

Comment:
It is not safe to just extend metrics, as metrics was passed externally (and therefore it can be quite unexpected for it to be mutated).
My original implementations copied metrics.

Reply:
Right, I'll add the copying part